### PR TITLE
reimplemented spiders

### PR DESCRIPTION
**Pull request by okushama**
unified egg entities/render
added poison blot render
reworked frog typings

### CODE REVIEW

**Comment by Cojomax99** [comment]
Looks good, but you'll need to update ItemMobEgg accordingly.

File: src/main/java/net/tropicraft/core/common/entity/hostile/EntityTreeFrog.java [hunk 1]
```diff
@@ -129,40 +132,39 @@ public void attackEntityWithRangedAttack(EntityLivingBase entity, float f) {
...
-	public void setType(byte b) {
-		this.getDataManager().set(TYPE, b);
+	public void setType(int i) {
+		this.getDataManager().set(TYPE, i);
 	}
 
 	@Override
 	public void writeEntityToNBT(NBTTagCompound n) {
-		n.setByte("frogType", getType().id);
+		n.setInteger("frogType", (byte)getType().ordinal());
```
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/entity/hostile/EntityTreeFrog.java]: setInteger(byte) :thinking:

File: src/main/java/net/tropicraft/core/common/item/ItemMobEgg.java [hunk 2]
```diff
@@ -102,10 +104,26 @@ public EnumActionResult onItemUse(ItemStack itemstack, EntityPlayer player, Worl
 	public static Entity spawnCreature(World worldIn, @Nullable String entityID, double x, double y, double z) {
 		if (entityID != null) {
 			Entity entity = null;
-
+			
+			EntityTreeFrog.Type frogType = null;
+			
+			if(entityID.endsWith("frog")) {
+				System.out.println("Ends in frog");
```
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/item/ItemMobEgg.java]: Can probably get rid of this sysout

**Review by Cojomax99 — APPROVED** [review]

**Comment by okushama** [comment]
PR/Issue #200 \o/

**Pull request merged**